examples/oracle_dsource/main.tf

```diff
@@ -0,0 +1,48 @@
+/**
+* Summary: This template showcases the properties available when creating an app data dsource.
+*/
+
+terraform {
+  required_providers {
+    delphix = {
+      version = "VERSION"
+      source  = "delphix-integrations/delphix"
+    }
+  }
+}
+
+provider "delphix" {
+  tls_insecure_skip = true
+  key               = "1.XXXX"
+  host              = "HOSTNAME"
+}
+
+
+
+resource "delphix_oracle_dsource" "test_oracle_dsource_second" {
```

Comment:
Old: `test_oracle_dsource_second`
New: `test_oracle_dsource` or `test_oracle_dsource_name`

Not sure what second is referring to.